src/chocolatey/docfx/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,17 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$packageName= 'DocFX'
+$toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$url        = 'https://github.com/dotnet/docfx/releases/download/v1.4/docfx.zip'

```

Comment:
This is a first pass, I imagine if you own this you can choose to embed the zip file directly in the package or use a variable for version to update this and the nuspec.